static/src/stylesheets/layout/nav/_veggie-burger.scss

```diff
@@ -10,7 +10,7 @@
     border-radius: 50%;
     outline: none;
     user-select: none;
-    z-index: 2;
+    z-index: $zindex-main-menu - 1;
```

Comment:
@zeftilldeath do you know if going from 2 to 1069 will be an issue? 

Could not see and problems and the button still interacts with the rest of the Nav the same way 😁